CHANGELOG.mdx

```diff
@@ -17,6 +17,9 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [3.1.0] - 2022-03-23                          No significant changes.
```

Comment:
🤔 this seems broken

Reply:
This is the SDK, there were no changlogs